integration_tests/src/main/python/window_function_test.py

```diff
@@ -455,3 +456,55 @@ def test_window_aggs_for_rows_collect_list():
             (partition by a order by b,c_int rows between CURRENT ROW and UNBOUNDED FOLLOWING) as collect_struct
         from window_collect_table
         ''')
+
+
+# Generates some repeated values to test the deduplication of GpuCollectSet.
+# And GpuCollectSet does not yet support struct type.
+_gen_data_for_collect_set = [
+    ('a', RepeatSeqGen(LongGen(), length=20)),
+    ('b', IntegerGen()),
```

Comment:
Could you make this a `LongRangeGen` instead, just so we know that there is no ambiguity to the sort order?

Reply:
Done.